qemu/tests/cfg/seabios_reboot_timeout.cfg

```diff
@@ -0,0 +1,27 @@
+- seabios_reboot_timeout:
+    virt_test_type = qemu
+    type = seabios_reboot_timeout
+    boot_menu = on
+    enable_sga = yes
+    images += ' stg'
+    image_name_stg = 'images/stg'
+    image_size_stg = 200M
+    force_create_image_stg = yes
+    remove_image_stg = yes
+    bootindex_stg = 0
+    bootindex_image1 = 1
+    variants:
+        - time0:
+            boot_reboot_timeout = 0
+        - time1000:
+            boot_reboot_timeout = 1000
+        - time10000:
+            boot_reboot_timeout = 10000
+        - time30000:
+            boot_reboot_timeout = 30000
+        - time60000:
+            boot_reboot_timeout = 60000
+        - time65535:
+            boot_reboot_timeout = 65535
+        - time65536:
+            boot_reboot_timeout = 65536
```

Comment:
Is this value set for limitation test? > 65535?
Then in .py file +38, you divided 1000, which means rb_timeout = 65, right?
It won't goes to line 41 > 65535 branch.
Do I understand your code correctly?

Reply:
Yes, you're right, will update it. Thanks!

Reply:
Updated, Thanks! @xiywang